core/pom.xml

```diff
@@ -34,19 +29,16 @@
             <dependency>
                 <groupId>org.junit.jupiter</groupId>
                 <artifactId>junit-jupiter-api</artifactId>
-                <version>${version.junit.jupiter}</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>org.junit.jupiter</groupId>
                 <artifactId>junit-jupiter-engine</artifactId>
-                <version>${version.junit.jupiter}</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>org.junit.vintage</groupId>
                 <artifactId>junit-vintage-engine</artifactId>
-                <version>${version.junit.vintage}</version>
                 <scope>test</scope>
             </dependency>
```

Comment:
You should remove these completely.  These dependencies are managed by the parent pom.  The child inherits the parent's dependency management, so if you had kept the versions in this would have been redundant.  Without the versions, I'm not sure what this accomplishes at all.  I think this is all that's in the dependency management section of this pom, so you can just remove dependency management completely.